File: PRO_LV1_소수찾기.py
def solution(n):
    answer = 0
    flag = 1
    
    for i in range(n):
        
        for j in range(i):
            target = i + 1
            tmp = j + 1
            
            if tmp != 1 and tmp != target:
                if target % tmp == 0:
                    flag = 1
                    break
                
        if flag == 1:
            flag = 0
            continue
        
        answer += 1
        
    return answer

File: PRO_LV1_소수찾기.java
//2022-07-25 시간초과
class Solution {
    public int solution(int n) {
        int answer = 0;
        for (int i = 1; i <= n; i++) {
            int mod = 0;
            int j = 2;
            while (j <= i) {
                if (i%j == 0) mod += 1;
                if (mod > 1) break;
                j += 1;
            }
            if (mod == 1) answer += 1;
        }
        return answer;
    }
}